marlowe/test/Spec/Marlowe/Common.hs

```diff
@@ -24,6 +28,11 @@ amount = integral (Range.constantFrom 0 (-100) 100)
 positiveAmount :: Gen Integer
 positiveAmount = integral (Range.constant 1 100)
 
+partyGen :: Gen Party
+partyGen = choice [ return $ Role "alice"
+                 , return $ PK (Ledger.pubKeyHash "6361726f6c")
```

Comment:
why isn't this generated?

Reply:
I want to merge marlowe-symbolic into marlowe next week. There are latest implementations of generators that Pablo did, and I want to reuse those everywhere, and here as well.
And for now it's good enough, I think: it tries both constructors, and names and actual hash doesn't matter mostly.